src/routes/cache-hooks/entities/event-type.entity.ts

```diff
@@ -1,15 +1,16 @@
 export const enum EventType {
   CHAIN_UPDATE = 'CHAIN_UPDATE',
-  NEW_CONFIRMATION = 'NEW_CONFIRMATION',
+  DELETED_MULTISIG_TRANSACTION = 'DELETED_MULTISIG_TRANSACTION',
   EXECUTED_MULTISIG_TRANSACTION = 'EXECUTED_MULTISIG_TRANSACTION',
-  PENDING_MULTISIG_TRANSACTION = 'PENDING_MULTISIG_TRANSACTION',
-  INCOMING_TOKEN = 'INCOMING_TOKEN',
   INCOMING_ETHER = 'INCOMING_ETHER',
+  INCOMING_TOKEN = 'INCOMING_TOKEN',
+  MESSAGE_CONFIRMATION = 'MESSAGE_CONFIRMATION',
+  MESSAGE_CREATED = 'MESSAGE_CREATED',
   MODULE_TRANSACTION = 'MODULE_TRANSACTION',
-  OUTGOING_TOKEN = 'OUTGOING_TOKEN',
+  NEW_CONFIRMATION = 'NEW_CONFIRMATION',
   OUTGOING_ETHER = 'OUTGOING_ETHER',
-  MESSAGE_CREATED = 'MESSAGE_CREATED',
-  MESSAGE_CONFIRMATION = 'MESSAGE_CONFIRMATION',
+  OUTGOING_TOKEN = 'OUTGOING_TOKEN',
+  PENDING_MULTISIG_TRANSACTION = 'PENDING_MULTISIG_TRANSACTION',
   SAFE_APPS_UPDATE = 'SAFE_APPS_UPDATE',
-  DELETED_MULTISIG_TRANSACTION = 'DELETED_MULTISIG_TRANSACTION',
+  SAFE_CREATED = 'SAFE_CREATED',
 }
```

Comment:
I'm just sorting the keys in alphabetical order here, besides adding `SAFE_CREATED`.